docs/sources/alerting/manage-notifications/create-contact-point.md

```diff
@@ -0,0 +1,58 @@
+---
+aliases:
+  - /docs/grafana/latest/alerting/contact-points/create-contact-point/
+  - /docs/grafana/latest/alerting/contact-points/message-templating/
+  - /docs/grafana/latest/alerting/message-templating/
+  - /docs/grafana/latest/alerting/unified-alerting/message-templating/
+  - /docs/grafana/latest/alerting/contact-points/delete-contact-point/
+  - /docs/grafana/latest/alerting/contact-points/edit-contact-point/
+  - /docs/grafana/latest/alerting/contact-points/test-contact-point/
+  - /docs/grafana/latest/alerting/manage-notifications/create-contact-point/
+keywords:
+  - grafana
+  - alerting
+  - guide
+  - contact point
+  - templating
+title: Manage contact points
+weight: 100
+---
+
+# Manage contact points
+
+Use contact points to define how your contacts are notified when an alert rule fires. You can create, edit, delete, and test a contact point.
+
+To add a contact point:
```

Comment:
this should be an H2 `Add a contact point` with a short intro, before you begin section (optional), stem sentence (optional), and then the steps.

Reply:
thanks Chris will update